### PR TITLE
Make listening ip:port explicit

### PR DESCRIPTION
**Pull request by darrenbilby**

### CODE REVIEW

File: tsctl [hunk 1]
```diff
@@ -130,4 +131,5 @@ if __name__ == '__main__':
     shell_manager.add_option(
         u'-c', u'--config', dest=u'config', default=u'/etc/timesketch.conf',
         required=False)
+    shell_manager.add_command(u'runserver', Server(host="127.0.0.1", port=5000))
```
Comment (berggren) [inline tsctl]: * Use single quotes and unicode for strings, "127.0.0.1" -> u'127.0.0.1'
* Move this line above the "add_option" to group together with the rest of the commands

**Comment by berggren** [comment]
Thanks for doing this! Just a couple of small nits, otherwise LGTM.

**Comment by darrenbilby** [comment]
thanks, updated

File: tsctl [hunk 2]
```diff
@@ -127,7 +128,8 @@ if __name__ == '__main__':
     shell_manager.add_command(u'add_user', AddUser())
     shell_manager.add_command(u'add_index', AddSearchIndex())
     shell_manager.add_command(u'drop_db', DropDataBaseTables())
+    shell_manager.add_command(u'runserver', Server(host='127.0.0.1', port=5000))
     shell_manager.add_option(
         u'-c', u'--config', dest=u'config', default=u'/etc/timesketch.conf',
         required=False)
-    shell_manager.run()
+        shell_manager.run()
```
Comment (berggren) [inline tsctl]: Indentation err :)

Comment (darrenbilby) [inline tsctl]: ugh, that'll teach me for using the web editor :)

**Pull request merged**